src/runtime_src/driver/xclng/drm/xocl/userpf/xocl_sysfs.c

```diff
@@ -51,21 +51,28 @@ static ssize_t user_pf_show(struct device *dev,
 }
 static DEVICE_ATTR_RO(user_pf);
 
-/* -live client contects-- */
+/* -live client contexts-- */
 static ssize_t kdsstat_show(struct device *dev,
 	struct device_attribute *attr, char *buf)
 {
 	struct xocl_dev *xdev = dev_get_drvdata(dev);
-	int size;
+	int size = 0;
 	xuid_t *xclbin_id;
+	pid_t *plist = NULL;
+	u32 clients, i;
 
 	xclbin_id = XOCL_XCLBIN_ID(xdev);
-	size = sprintf(buf,
-			   "xclbin:\t\t\t%pUb\noutstanding execs:\t%d\ntotal execs:\t\t%ld\ncontexts:\t\t%d\n",
-			   xclbin_id ? xclbin_id : 0,
-			   atomic_read(&xdev->outstanding_execs),
-			   atomic64_read(&xdev->total_execs),
-			   get_live_client_size(xdev));
+	size += sprintf(buf + size, "xclbin:\t\t\t%pUb\n",
+		xclbin_id ? xclbin_id : 0);
+	size += sprintf(buf + size, "outstanding execs:\t%d\n",
+		atomic_read(&xdev->outstanding_execs));
+	size += sprintf(buf + size, "total execs:\t\t%ld\n",
+		atomic64_read(&xdev->total_execs));
```

Comment:
I wonder if should keep the original contexts and then add the pids in the following line?